CHANGELOG.rst

```diff
@@ -7,6 +7,9 @@ in development
 Fixed
```

Comment:
If this PR enhancement is not a bugfix, but improvement or change, it should come under different category.

See `CHANGELOG.rst` history for other examples.